### PR TITLE
Guidelines

### PR DESCRIPTION
**Pull request by rakvat**
I took content from vinyl and manually converted it to markdown.

### CODE REVIEW

File: guidelines/community/en.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Community Guidelines
+
+Our number one priority is to create a fun and safe place for kinksters, like you and I, to call home here on the interwebs.
+
+To make sure FetLife is a fun and safe place, we wrote the following list of general community guidelines we would like everyone to respect.
+
+Just remember, that your participation on FetLife is always subject to our [Terms of Use](https://fetlife.com/legalese/tou).
```
Comment (philippt) [inline guidelines/community/en.md]: I believe there should be no comma behind "remember"

File: guidelines/community/en.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+## Don't Out Anyone
+
+Each person on FetLife is comfortable with different pieces of their personal information being revealed. Some people don't mind you using their full real name and others don't want you to even use their first name. Please make a conscious effort to respect each person's comfort level.
+
+Also, to protect members of the community, we don't allow phone numbers or addresses to be posted publicly on FetLife unless they are for a business, organization or event.
+
+## Take Everything You Read With A Grain of Salt
+
+FetLife is for entertainment, informational, and educational purposes only. What people post is not verified by anyone for it's accuracy. What you read on FetLife should not be construed as actual, professional advice. You should always consult a qualified professional.
```
Comment (philippt) [inline guidelines/community/en.md]: its accuracy

File: guidelines/content/en.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Crushing food and inanimate objects is ok. I am not a big fan of hunting. But I do watch Duck Dynasty&hellip; Jack.
+
+---
+
+Also, in general, we think that intellectual discussions, jokes between friends, and private discussions should all be fair game.
+
+Our guidelines aren’t black and white nor do we think it's healthy for them to be. For every guideline there are tens of thousands of edge cases that we will have to review on a case by case basis and act on them by taking into consideration multiple factors including context, how the average person might interpret something, etc.
+
+If at any point you're ever concerned something you're posting might be misconstrued the wrong way then please make the extra effort to assure things are clear as possible.
```
Comment (philippt) [inline guidelines/content/en.md]: "as clear as possible"?

File: guidelines/group_leader/en.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+Just remember, participation on FetLife is always subject to our [Terms of Use](https://fetlife.com/legalese/tou).
+
+## See a Need, Fill a Need, Start a Group
+
+Have a new idea for a group? Can't find a group that speaks to you? Want to start a group to promote your business, event or organization. Then just do it already. [Start a new group](https://fetlife.com/groups/new).
+
+## Love Your Group
+
+The fun part is getting your group off the floor. A little bit of time and love will go along way in creating a successful group. Here are some tips:
```
Comment (philippt) [inline guidelines/group_leader/en.md]: a long way

File: guidelines/group_leader/en.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+> “I personally have no problems with camel pitching. I think it is natural to like pitching camels because I think their humps were made for pitching as explained in Camel Pitching for Dummies pages 69-73.”
+
+&hellip;is ok. Sharing fantasies, stories and looking for others into camel pitching (where camel pitching is one of the topics that is a no-no) is not ok.
+
+## Don't Let People Ruin It For Others
+
+We want you to encourage healthy debate but there is a big difference between that and a discussion which has degraded into people attacking one another.
+
+Aggressive personal attacks, criminal accusations, making fun of, trolling, flaming, bullying, racial slurs or any other type of malicious behaviour that lowers the quality of a discussion needs to be moderated quickly before things get out of control and ruins the experience for everyone. Ex:
```
Comment (philippt) [inline guidelines/group_leader/en.md]: ruins -> ruin ?

**Comment by spilth** [comment]
👍

**Comment by johnbaku** [comment]
@philippt should have been an English teacher!!! Impressive.

@rakvat I am cool with all the suggested changed @philippt made.

**Comment by reneklacan** [comment]
> @rakvat I am cool with all the suggested changed @philippt made.

`changed` -> `changes` :trollface:

**Comment by johnbaku** [comment]
@reneklacan bully! :-(

**Pull request merged**